job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy

```diff
@@ -37,6 +38,7 @@ pipeline{
                         [\$class: 'StringParameterValue', name: 'KIE_GROUP_DEPLOYMENT_REPO_URL', value: 'https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/service/local/repositories/scratch-release-rhba-main/content-compressed'],
                         [\$class: 'StringParameterValue', name: 'UMB_VERSION', value: 'main'],
                         [\$class: 'StringParameterValue', name: 'PRODUCT_VERSION', value: '7.12.0'],
+                        [\$class: 'StringParameterValue', name: 'CONFIG_BRANCH', value: "\${env.MAIN_CONFIG_BRANCH}"],
```

Comment:
i asked if `DEFAULT_CONFIG_BRANCH` is better name for the parameter alson in Jenkins build config ;)

Reply:
Changed! Thanks!